backend/coins/btc/handlers/handlers.go

```diff
@@ -101,11 +102,25 @@ func (handlers *Handlers) formatAmountAsJSON(amount coin.Amount, isFee bool) For
 	}
 }
 
+func (handlers *Handlers) formatAmountAtTimeAsJSON(amount coin.Amount, isFee bool, timeStamp *time.Time) FormattedAmount {
```

Comment:
remove the isFee param, as it is always `false`?

or is the plan to also add the fee fiat at the time of transaction?

Reply:
Good point! I don't think we are planning to add historical fiat value for the fee. I'll just remove it.